projects/storefrontapp/src/environments/productconfig/productconfig.feature.ts

```diff
@@ -1,11 +1,28 @@
+import { translationChunksConfig, translations } from '@spartacus/assets';
 import { CommonConfiguratorModule } from '@spartacus/product/configurators/common';
 import { TextfieldConfiguratorModule } from '@spartacus/product/configurators/textfield';
 import { VariantConfiguratorModule } from '@spartacus/product/configurators/variant';
+import {
+  configuratorTranslationChunksConfig,
+  configuratorTranslations,
+} from 'feature-libs/product/configurators/common/assets/public_api';
 import { FeatureEnvironment } from '../models/feature.model';
 export const productConfigFeature: FeatureEnvironment = {
   imports: [
-    TextfieldConfiguratorModule,
-    CommonConfiguratorModule,
-    VariantConfiguratorModule,
+    CommonConfiguratorModule.withConfig({
+      i18n: {
+        resources: {
+          en: { ...translations.en, ...configuratorTranslations.en },
```

Comment:
It's only feature.ts file (it's not released, it's only for our development process), so let's leave it as is. however I can see we can improve in future (also for my org) not to reference main translations in all feature.ts files, but rather only in app.module.
Then the config mechanism will merge those main translations with any other feature-translations deeply anyway ( by desing of the config mechanism - it deep merges all the objects).